
package com.divanov.mathwebservice.gen;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.5.3
 * 2023-03-08T16:22:01.058+03:00
 * Generated source version: 3.5.3
 */

@WebFault(name = "solveQuadraticEducationException", targetNamespace = "http://math.ws.divanov")
public class SolveQuadraticEducationException extends Exception {

    private com.divanov.mathwebservice.gen.SolveQuadraticEducationExceptionDetail faultInfo;

    public SolveQuadraticEducationException() {
        super();
    }

    public SolveQuadraticEducationException(String message) {
        super(message);
    }

    public SolveQuadraticEducationException(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public SolveQuadraticEducationException(String message, com.divanov.mathwebservice.gen.SolveQuadraticEducationExceptionDetail solveQuadraticEducationException) {
        super(message);
        this.faultInfo = solveQuadraticEducationException;
    }

    public SolveQuadraticEducationException(String message, com.divanov.mathwebservice.gen.SolveQuadraticEducationExceptionDetail solveQuadraticEducationException, java.lang.Throwable cause) {
        super(message, cause);
        this.faultInfo = solveQuadraticEducationException;
    }

    public com.divanov.mathwebservice.gen.SolveQuadraticEducationExceptionDetail getFaultInfo() {
        return this.faultInfo;
    }
}
